src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,54 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MINIMUM_USER_AGE = 18;
+    private static final int MINIMUM_PASSWORD_LENGTH = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
+        user = checkUserNotNull(user);
+        user = checkUserLogin(user);
+        user = checkUserAge(user);
+        user = checkUserPassword(user);
+        if (user != null) {
+            storageDao.add(user);
+            return user;
+        }
```

Comment:
```suggestion
        storageDao.add(user);
        return user;
```
You don't need this check because you already have it in the checkUserNotNull method